cli/compose/convert/service.go

```diff
@@ -217,13 +217,17 @@ func convertServiceSecrets(
 		if gid == "" {
 			gid = "0"
 		}
+		mode := secret.Mode
+		if mode == 0 {
```

Comment:
Just curious, is there a situation one would want mode 0 for? It would not be readable or writeable by any user including `root` so to me that would be kind of like setting `--name` for a container to an empty string.